common/static/js/capa/jsinput/jsinput_example.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Dropdown with Dynamic Text</title>
+    <link rel="stylesheet" type="text/css" href="jsinput_example.css">
```

Comment:
jsinput_example.css and jsinput_example.js will be changed to files.edx.org full URLs once they are up.